cilium/cmd/monitor.go

```diff
@@ -44,7 +44,7 @@ const (
 // monitorCmd represents the monitor command
 var monitorCmd = &cobra.Command{
 	Use:   "monitor",
-	Short: "Monitoring",
```

Comment:
The consumable is consumed by consumers... the monitor provides monitoring for monitors.
More seriously, it's probably obvious but "display endpoint BPF program network events" might help illustrate where in the stack this is getting it's data (and avoid using "monitor" again). This is a bike-shedding nit, however.

Reply:
Monitoring is fairly self-evident though - you are monitoring BPF program events as they occur, vs. the abstract nature of a `Consumable` ;)

Reply:
I agree though, I don't like self-referential definitions. Will fix.

Reply:
Fixed in latest upload :)

Reply:
hehe, yeah, I didn't think what I said was particularly better. +1 to whatever you decide.